Makefile

```diff
@@ -313,6 +327,7 @@ UNIT_TEST_PATHS := ./controller/... ./internal/... ./pkg/... ./modules/...
 
 .PHONY: _test.unit
 _test.unit: gotestsum
+	KUBEBUILDER_ASSETS="$(shell $(SETUP_ENVTEST) use -p path)" \
```

Comment:
This shouldn't be necessary given that we don't need `envtest` in UTs.

```suggestion
```

Reply:
done.